.github/workflows/trivy.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: trivy
+
+on:
+  push:
+    branches: [ "main", "develop" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
```

Comment:
Maybe we could review develop branch only and create a PR to develop.

Reply:
Yes, I would just launch a PR against develop